adb_graphics/datahandler/grib.py

```diff
@@ -451,6 +576,14 @@ def grid_info(self):
 
         return grid_info
 
+    def run_total(self, values, **kwargs) -> np.ndarray:
+
+        ''' Sums over all the forecast lead times available. '''
+
+        # pylint: disable=unused-argument,no-self-use
```

Comment:
This one had a conflict between the test, which needed the callable function to be a true method, as opposed to the linter suggesting that since we aren't using "self" it could be a static method. I went with the ability to test, and quieted the linter.